modules/EvManager/main/car_simulatorImpl.cpp

```diff
@@ -0,0 +1,586 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "car_simulatorImpl.hpp"
+#include "main/SimCommand.hpp"
+#include <everest/logging.hpp>
+
+namespace module::main {
+
+void car_simulatorImpl::init() {
+    loopIntervalMs = defaultLoopIntervalMs;
```

Comment:
Adding a `constexpr` here instead of `defaultLoopIntervalMs`. E.g. `constexpr auto DEFAULT_LOOP_INTERVAL_MS = 250;`

Reply:
Makes sense. I also removed the optional